README.md

```diff
@@ -74,19 +74,23 @@ In this section we'll use the Azure Portal to create the required resources to h
 
 ### Configuring the Resources
 
-Once all the resources are created, you will need to get the connection information for the Postgres and Storage account to the Web App, as well as configure the resources for use.
+Once all the resources are created, you will need to get the connection information for the Postgres and Storage account to the Web App (App Service), as well as configure the resources for use.
 
 #### Configure the Storage Account
 
-1. Navigate to the Storage Account resource, then **Data storage** - **Containers**
+1. Navigate to the **Storage Account** resource, then **Data storage** - **Containers**
 1. Create a new Container, provide a _Name_, `keystone-uploads`, and set _Public access level_ to `Blob`, then click **Create**
 1. Navigate to **Security + networking** - **Access keys**, copy the _Storage account name_ and _key1_
-1. Navigate to the **Web App** you created and go to **Settings** - **Configuration**
-1. Create new application settings for the Storage account, storage account key and container name (these will become the environment variables available to Keystone) and click _Save_
+1. Navigate to the **Web App (App Service)** you created and go to **Settings** - **Configuration**
+1. Create three **New application settings**:
+   - `AZURE_STORAGE_ACCOUNT` as the _Storage account name_ value you copied above.
+   - `AZURE_STORAGE_KEY` as the _key1_ value you copied above.
+   - `AZURE_STORAGE_CONTAINER` as the name of the container, which you specified as `keystone-uploads` above.
+1. These will become the environment variables available to Keystone, click _Save_.
 
 #### Configure Postgres
 
-1. Navigate to the Postgres resource then **Settings** - **Connection security**
+1. Navigate to the **Azure Database for PostgreSQL server** resource then **Settings** - **Connection security**
```

Comment:
Renaming this specific one only, other references are just Postgres, as this one is literally what it's called to be clearer.